tests/e2e/travis-e2e.sh

```diff
@@ -13,7 +13,7 @@ chmod +x kubectl
 curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/v0.6.1/kind-linux-amd64
 chmod +x kind
 
-./kind create cluster --image=kindest/node:v1.17.0
+./kind create cluster --image=kindest/node:v1.18.1
```

Comment:
You should use `v1.18.0` instead. The 1.18.1 image as not yet been pushed to [dockerhub](https://hub.docker.com/r/kindest/node/tags?page=1&name=1.18).

Reply:
This is why I like minikube more :D sorry kind folks